docs/content/docs/recipes/collaborative-text-area.md

```diff
@@ -0,0 +1,325 @@
+
+The next image shows the same four clients after an edit was made in one of the browsers. Note that the text has updated in all four browsers.
+
+![Four browsers with the Timestamp app open in them one second after the button has been pushed.](https://fluidframework.blob.core.windows.net/static/images/collaborative_text_area_2.png)
+
+{{< callout note >}}
+
+This tutorial assumes that you are familiar with the [Fluid Framework Overview]({{< relref "/docs/_index.md" >}}) and that you have completed the [Quick Start]({{< relref "quick-start.md" >}}). You should also be familiar with the basics of [React](https://reactjs.org/), [creating React projects](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app), and [React Hooks](https://reactjs.org/docs/hooks-intro.html).
+
+{{< /callout >}}
+
+## Create the project
+
+1. Open a Command Prompt and navigate to the parent folder where you want to create the project; e.g., `c:\My Fluid Projects`.
+1. Run the following command at the prompt. (Note that the CLI is np**x**, not npm. It was installed when you installed Node.js.)
+
+    ```dotnetcli
+    npx create-react-app collaborative-text-area-tutorial --use-npm
+    ```
+
+1. The project is created in a subfolder named `collaborative-text-area-tutorial`. Navigate to it with the command `cd fluid-react-tutorial`.
+1. The project uses two Fluid libraries:
+
+    |Library |Description |
+    |---|---|
+    | `fluid-framework`    |Contains the SharedMap and SharedString [distributed data structures]({{< relref "dds.md" >}}) that synchronize data across clients. *This object will hold the most recent timestamp update made by any client.*|
+    | `@fluidframework/tinylicious-client`   |Defines the connection to a Fluid server and defines the starting schema for the [Fluid container]({{< relref "containers.md" >}}).|
+    {.table}
+
+    Run the following command to install the libraries.
+
+    ```dotnetcli
+    npm install @fluidframework/tinylicious-client fluid-framework
+    ```
+
+## Code the project
+
+1. Open the file `\src\App.js` in your code editor. Delete all the default `import` statements except the one that imports `App.css`. Then delete all the markup from the `return` statement. The file should look like the following:
+
+    ```js
+    import "./App.css";
+
+    function App() {
+      return (
+
+      );
+    }
+
+    export default App;
+    ```
+
+1. Add the following `import` statements. Note: `CollaborativeTextArea` will be defined later.
+
+    ```js
+    import React from "react";
+    import { TinyliciousClient } from "@fluidframework/tinylicious-client";
+    import { SharedString } from "fluid-framework";
+    import { CollaborativeTextArea } from "./CollaborativeTextArea";
+    import { SharedStringHelper } from "./SharedStringHelper";
```

Comment:
this import got updated in the finished example, right?  it should be updated through here too